tests/test_flight.py

```diff
@@ -594,3 +618,224 @@ def test_hybrid_motor_flight(mock_show, calisto_hybrid_modded):
+    flight_calisto_robust, flight_calisto_nose_to_tail, atol=1e-8
+):
+    """Tests the effective rail length of the flight simulation. The expected
+    values are calculated by hand, and should be valid as long as the rail
+    length and the position of the buttons and nozzle do not change in the
+    fixtures. If the fixtures change, this test must be updated. It is important
+    to keep
+
+    Parameters
+    ----------
+    flight_calisto_robust : rocketpy.Flight
+        Flight object to be tested. See the conftest.py file for more info
+        regarding this pytest fixture.
+    flight_calisto_nose_to_tail : rocketpy.Flight
+        Flight object to be tested. The difference here is that the rocket is
+        defined with the "nose_to_tail" orientation instead of the
+        "tail_to_nose" orientation. See the conftest.py file for more info
+        regarding this pytest fixture.
+    atol : float, optional
+        The absolute tolerance error, by default 1e-8
+    """
+    test1 = flight_calisto_robust
+    test2 = flight_calisto_nose_to_tail
+
+    rail_length = 5.2
+    upper_button_position = 0.082
+    lower_button_position = -0.618
+    nozzle_position = -1.373
+
+    effective_1rl = rail_length - abs(upper_button_position - nozzle_position)
+    effective_2rl = rail_length - abs(lower_button_position - nozzle_position)
+
+    # test 1: Rocket with "tail_to_nose" orientation
+    assert pytest.approx(test1.effective_1rl, abs=atol) == effective_1rl
+    assert pytest.approx(test1.effective_2rl, abs=atol) == effective_2rl
+    # test 2: Rocket with "nose_to_tail" orientation
+    assert pytest.approx(test2.effective_1rl, abs=atol) == effective_1rl
+    assert pytest.approx(test2.effective_2rl, abs=atol) == effective_2rl
+
+
+def test_max_values(flight_calisto_robust, atol=5e-3):
+    """Test the max values of the flight. This tests if the max values are
+    close to the expected values. However, the expected values were NOT
+    calculated by hand, it was just copied from the test results. This is
+    because the expected values are not easy to calculate by hand, and the
+    results are not expected to change. If the results change, the test will
+    fail, and the expected values must be updated. If if want to update the
+    values, always double check if the results are really correct. Acceptable
+    reasons for changes in the results are: 1) changes in the code that
+    improve the accuracy of the simulation, 2) a bug was found and fixed. Keep
+    in mind that other tests may be more accurate than this one, for example,
+    the acceptance tests, which are based on the results of real flights.
+
+    Parameters
+    ----------
+    flight_calisto_robust : rocketpy.Flight
+        Flight object to be tested. See the conftest.py file for more info
+        regarding this pytest fixture.
+    atol : float, optional
+        Absolute tolerance error, by default 5e-3
+    """
+    test = flight_calisto_robust
```

Comment:
I believe `robust_flight` is not the best name because I think in the future we could try adding other rockets beyond calisto. That's why I wanted to preserve the names

Reply:
Besides my last comment, I have two more reasons for keeping the
* Not the end of the world using the name "test", as long as you clearly define what this test is (test = xxxx)
* Hopefully one day this test can be used for more than one flight object, like in a parametrize fixture scheme.

An alternative, though, is to use the simple name "flight".

I'm against dropping the "calisto" name from the variable. More rockets may be tested in the future.